### DISCUSSION TITLE
Line items - taxes

### DISCUSSION THREAD

**Discussion by GarethHumbel** [YetiForceCompany/YetiForceCRM#16160] [General] [answered]
In my taxing scenario, I need to apply two taxes at the same time: +VAT and -OTHER TAX
It means that VAT is added to TOTAL NET (as everywhere is) but OTHER TAX is deducted from TOTAL NET
What would you advise in this scenario?

**Comment by s2b-git** [comment]
It would need to allow negative numbers. Probably lots of dependencies linked to other inventory modules

**Comment by mariuszkrzaczkowski** [comment, marked as answer]
I pre-checked and it looks like there aren't many changes to make, check out:

1.  https://gitdeveloper.yetiforce.com/index.php?module=Inventory&parent=Settings&view=TaxConfiguration
![image](https://user-images.githubusercontent.com/10198654/173905199-06cacd45-822d-4e11-bfd7-e9f6f081fce2.png)
2. change validations
```
data-validation-engine="validate[required]"
```
https://github.com/YetiForceCompany/YetiForceCRM/blob/b8709bc012b91ca949ecb865b32c2c7b550e0b72/layouts/basic/modules/Vtiger/InventoryTaxesType2.tpl#L14
3.
![image](https://user-images.githubusercontent.com/10198654/173905684-788449d0-718b-40aa-a23c-f341ec6585cb.png)

**Reply by ArniSamano** [reply]
Sorry to bother... I have similar thing with negative value, but I want to set this negative tax in the preset tax list. Despite removing validation  as per your suggestion, I still cannot add negative tax. Entering it as a custom tax each time line-item module is edited may lead to many human errors (that's why I would like to add this negative tax to the global list)
